django_sorcery/db/composites.py

```diff
@@ -67,8 +70,8 @@ def __init__(self, *args, **kwargs):
             setattr(self, k, v)
 
     @classproperty
-    def _columns(cls):
```

Comment:
should stay as `cls` as its a class property. `self` is never passed as in class instance. class itself is passed. usually convention is to call it `cls`. see https://docs.python.org/3.7/library/functions.html#classmethod

Reply:
yes, sadly pylint is tripping over this because it doesn't understand `classproperty`